test/unit/foreman/renderer/snapshots.yaml

```diff
@@ -1,71 +1,260 @@
 # Edit this file and then run: rake snapshots:generate RAILS_ENV=test
 
 files:
-  - provision/alterator_default.erb
-  - cloud_init/cloud_init_default.erb
-  - PXELinux/pxelinux_global_default.erb
-  - PXEGrub/pxegrub_global_default.erb
-  - PXEGrub2/pxegrub2_global_default.erb
-  - PXELinux/pxelinux_default_local_boot.erb
-  - PXEGrub/pxegrub_default_local_boot.erb
-  - PXEGrub2/pxegrub2_default_local_boot.erb
-  - PXELinux/pxelinux_default_memdisk.erb
-  - PXELinux/pxelinux_chain_ipxe.erb
-  - PXELinux/pxelinux_chain_ipxe_undi.erb
-  - provision/alterator_default.erb
-  - finish/alterator_default_finish.erb
-  - provision/atomic_kickstart_default.erb
-  - provision/autoyast_default.erb
-  - provision/autoyast_sles_default.erb
-  - PXELinux/autoyast_default_pxelinux.erb
-  - user_data/autoyast_default_user_data.erb
-  - user_data/userdata_open_vm_tools.erb
-  - provision/coreos_provision.erb
-  - PXELinux/coreos_pxelinux.erb
-  - finish/freebsd_(mfsbsd)_finish.erb
-  - provision/freebsd_(mfsbsd)_provision.erb
-  - PXELinux/freebsd_(mfsbsd)_pxelinux.erb
-  - provision/jumpstart_default.erb
-  - provision/junos_default_slax.erb
-  - ZTP/junos_default_ztp_config.erb
-  - finish/junos_default_finish.erb
-  - provision/kickstart_default.erb
-  - finish/kickstart_default_finish.erb
-  - PXELinux/kickstart_default_pxelinux.erb
-  - PXEGrub/kickstart_default_pxegrub.erb
-  - PXEGrub2/kickstart_default_pxegrub2.erb
-  - user_data/kickstart_default_user_data.erb
-  - POAP/nx-os_default_poap_setup.erb
-  - provision/preseed_default.erb
-  - finish/preseed_default_finish.erb
-  - PXELinux/preseed_default_pxelinux.erb
-  - PXEGrub2/preseed_default_pxegrub2.erb
-  - user_data/preseed_default_user_data.erb
-  - PXELinux/rancheros_pxelinux.erb
-  - provision/rancheros_provision.erb
-  - user_data/userdata_default.erb
-  - PXELinux/waik_default_pxelinux.erb
-  - provision/xenserver_default_answerfile.erb
-  - finish/xenserver_default_finish.erb
-  - PXELinux/xenserver_default_pxelinux.erb
-  - iPXE/ipxe_intermediate_script.erb
-  - snippet/ansible_tower_callback_script.erb
-  - snippet/ansible_tower_callback_service.erb
-  - snippet/chef_client.erb
-  - snippet/coreos_cloudconfig.erb
-  - snippet/epel.erb
-  - snippet/fix_hosts.erb
-  - snippet/freeipa_register.erb
-  - snippet/preseed_networking_setup.erb
-  - snippet/puppet.conf.erb
-  - snippet/puppet_setup.erb
-  - snippet/pxegrub2_chainload.erb
-  - snippet/pxegrub2_discovery.erb
-  - snippet/pxegrub_chainload.erb
-  - snippet/pxegrub_discovery.erb
-  - snippet/pxelinux_chainload.erb
-  - snippet/pxelinux_discovery.erb
-  - snippet/rancheros_cloudconfig.erb
-  - snippet/remote_execution_ssh_keys.erb
-  - snippet/saltstack_minion.erb
-  - snippet/saltstack_setup.erb
+# global boot
+  - 
+    name: PXELinux/pxelinux_global_default.erb
+    os: []
```

Comment:
Can we get this from the template metadata?

Reply:
Well only operating system name and version. Not family and possibly other things. I'd rather be explicit. I am currently working on adding host params here.

Reply:
Well, I want specifically to be testing RHEL 6, 7, 8 and similarly Debian/Ubuntu. Let me think about it tomorrow.